### DISCUSSION TITLE
How is recorder_manager used?

### DISCUSSION THREAD

**Discussion by H-Hisamichi** [isaac-sim/IsaacLab#1582] [Q&A]
Hello All,

Where should the new `recorder_manager` feature be coded in the RL code?
Define as a new class in `MyRL_env_cfg.py`?

Thank you!

**Comment by H-Hisamichi** [comment]
Dear All,

Coding the following code in `MyRL_env_cfg.py` worked. If there are any mistakes, please point them out.
my code:
```
@configclass
class RecordersCfg(RecorderManagerBaseCfg):
    """Recorders terms for the MDP."""
    dataset_export_dir_path="/home/robot/IsaacLab_v1.4.0/IsaacLab/logs/recorder_test_logs"
    post_step_states = mdp.PostStepStatesRecorderCfg()
```

Also, this code recorded joint position, joint velocity, root pose, and root velocity. Can I record additional joint torque?

Thank you!

**Comment by H-Hisamichi** [comment]
I created a new Recorder term to implement this feature.
```
class PostStepStatesANDJointTorquesRecorder(RecorderTerm):
    """Recorder term for state and joint torques."""
    def record_post_step(self, asset_cfg: SceneEntityCfg = SceneEntityCfg("robot")):
        states = self._env.scene.get_state(is_relative=True)
        asset: Articulation = self._env.scene[asset_cfg.name]
        joint_torques = asset.data.applied_torque[:, asset_cfg.joint_ids]
        states['articulation']['robot']['joint_torques'] = joint_torques
        return "states", states
```
Thanks for watching!

**Reply by Kt981** [reply]
Hi, this script can replace the recorder manager?

**Reply by H-Hisamichi** [reply]
Hi @Kt981,

This script alone does not work, but it can extend the recorder functionality when added as a `Recorder Manager MDP`.
